ConsumerApi.Tests.Integration/API/ChallengesApi.cs

```diff
@@ -10,11 +9,11 @@ public class ChallengesApi : BaseApi
 
     public async Task<HttpResponse<Challenge>> CreateChallenge(RequestConfiguration requestConfiguration)
     {
-        return await ExecuteRequest<Challenge>(Method.Post, new PathString(ROUTE_PREFIX).Add("/challenges").ToString(), requestConfiguration);
+        return await Post<Challenge>("/challenges", requestConfiguration);
```

Comment:
Just noticed that the route parts are lowercase in the tests. But in the Backbone, they have an uppercase first latter (=> "/Challenges"). Can you do a quick scan of the tests and fix that please?